src/Ticket.php

```diff
@@ -1547,9 +1558,22 @@ public function computeTakeIntoAccountDelayStat()
             isset($this->fields['id'])
             && !empty($this->fields['date'])
         ) {
-            $calendars_id = $this->getCalendar();
+            $calendars_id = 0;
             $calendar     = new Calendar();
-
+	    // Not clear how to handle case with both SLA and OLA
+	    // For now prefer OLA calendar
+	    if (isset($this->fields['olas_id_tto']) && $this->fields['olas_id_tto'] > 0){
+		    $la = new OLA();
+		    $la->getFromDB($this->fields['olas_id_tto']);
+	    } elseif (isset($this->fields['slas_id_tto']) && $this->fields['slas_id_tto'] > 0){
+		    $la = new SLA();
+		    $la->getFromDB($this->fields['slas_id_tto']);
+	    }
+	    if (isset($la) && $la->fields['use_ticket_calendar']){
+		    $calendars_id = $la->getField('calendars_id');
+	    } else {
+		    $calendars_id = parent::getCalendar();
+	    }
```

Comment:
`takeintoaccount_delay_stat` reflects the "customer" point of view, and it should not be changed.
We have already identified that this internal "take into account" delay information is missing, but it should probably be store in a new `internal_takeintoaccount_delay_stat` field. Also, this information may be sensitive, from a commercial point of view, so I do not know how and where it should be displayed.

This should be changed in another PR, to keep small PRs that would be easier to validate.

Reply:
Please check below, I've updated code to use just SLA TTO calendar (if defined) and fallback to previous call of Ticket::getCalendar() if it is not.

Reply:
Ok, I've reverted the fixes to computeTakeIntoAccountDelayStat
I wanted to combine it, to have takeintoaccount_delay_stat correct after this PR (we have a case when SLA::TTO/TTR are using different calendars with different working hours)
I'll open a separate PR for that

Reply:
Submitted PR #12646 for fixing computeTakeIntoAccountDelayStat